edk2toolext/edk2_logging.py

```diff
@@ -255,6 +255,13 @@ def __init__(self):
         self._verbose = False
         self._currentSection = "root"
 
+        secrets_regex_strings = [
+            r"[A-Za-z0-9]{46}",  # Nuget API Key
+            r"github_pat_[^ ]*"  # Github PAT
```

Comment:
I think it would be better to go with what GitHub has strictly allowed as PAT characters (`[A-Za-z0-9_]`).

For example, right now `<PAT>,<other data>` would include the text after the comma though the comma would end the PAT (just as much as a space) per GitHub's allowed character set.

Reply:
The documentation you sent was great. I've updated to match and added a few tests.